Store format page edits (#1)
renetapopova/docs-operations: modules/ROOT/pages/database-internals/store-formats.adoc

= Store formats
:description: This page describes store formats in Neo4j.

Neo4j's storage engine supports several store formats that describe how data is written to disk. 
Starting from 5.23, the `block` format is the recommended format for Enterprise Edition due to its superior performance and scalability.
This `block` format uses advanced data structures and inlining techniques to enhance data locality, which leads to better resource utilization.

`aligned` is the recommended format for Community Edition.

`standard` and `high_limit` formats are deprecated starting from Neo4j 5.23. 
It is not recommended to use these formats for new databases.

> For more information on the deprecation and timeline for eventual removal of these formats, see <<record-format-deprecation, Record format deprecation>>.

Reviewer: ```suggestion
For more information on the deprecation and timeline for eventual removal of these formats, see <<format-deprecations, Format deprecations>>.
```

* For information on deprecation and eventual removal, see <<record-format-deprecations, Record format deprecations>>.

> [[record-format-deprecations]]

Reviewer: ```suggestion
[[format-deprecations]]
```

== Format deprecations

`standard` and `high_limit` formats are deprecated starting from Neo4j 5.23.

`block` format provides a better performance and scalability than the deprecated formats. 
They will remain part of the product throughout the v5 and vNext server lifecycles. 

> The last version of Neo4j to include these formats will be vNext.LTS which is planned for release in November 2026. LTS versions of Neo4j are supported for 3 years following their release. 

Reviewer: ```suggestion
The last version of Neo4j to include these formats will be vNext.LTS which is planned for release in November 2026. 
LTS versions of Neo4j are supported for 3 years following their release. 
```

This means that support for `standard` and `high_limit` formats is planned to end in November 2029.

> It is recommended that Enterprise Edition users migrate all databases to block format at their earliest convenience to ensure the best possible performance.

Reviewer: ```suggestion
It is recommended that Enterprise Edition users migrate all databases to `block` format at their earliest convenience to ensure the best possible performance.
```

For more information, see <<change-store-format, Changing the store format of existing databases>>.

=== FAQ

. *How can I find out the format of my database?* +
You can use the `SHOW DATABASES YIELD name, store` Cypher command to get information about the store format of a database. 

> See https://neo4j.com/docs/operations-manual/current/database-administration/standard-databases/listing-databases/#_show_detailed_information_for_a_particular_database

Reviewer: ```suggestion
See xref:database-administration/standard-databases/listing-databases.adoc#_show_detailed_information_for_a_particular_database[Show detailed information for a particular database] for a detailed example.
```

. *When will support for `standard` and `high_limit` formats end?* +

> The last version of Neo4j to include these formats will be an LTS release planned for release in November 2026. 

Reviewer: ```suggestion
The last version of Neo4j to include these formats will be an LTS release planned for November 2026, which will be supported for three years until November 2029.
```

> This LTS release will be supported for three years, so November 2029.

Reviewer: ```suggestion
```
